lib/runner/test-runners/cucumber.js

```diff
@@ -246,6 +246,10 @@ class CucumberSuite extends TestSuite {
 
 class CucumberRunnner extends Runner {
   get supportsConcurrency() {
+    return true;
+  }
+
+  get supportsTestWorkers() {
```

Comment:
`suppportsTestWorkers` seems confusing (the term "test workers" is used at different places in Nightwatch in different contexts). I think we should instead change the name to `supportsParallelTestSuiteRun`, which exactly tells what we are trying to do here (what we are disabling in Cucumber).

Reply:
updated as suggested